webapp/javascript/pages/continuous/contextMenu/AnnotationInfo.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { Dispatch, SetStateAction } from 'react';
+import { MenuItem, applyStatics } from '@webapp/ui/Menu';
+import { Popover, PopoverBody, PopoverFooter } from '@webapp/ui/Popover';
+import Button from '@webapp/ui/Button';
+import { Portal } from '@webapp/ui/Portal';
+import TextField from '@webapp/ui/Form/TextField';
+import { AddAnnotationProps } from './AddAnnotation.menuitem';
+import { useAnnotationForm } from './useAnnotationForm';
+
+interface AnnotationInfo {
+  container: AddAnnotationProps['container'];
+  popoverAnchorPoint: AddAnnotationProps['popoverAnchorPoint'];
+  timestamp: AddAnnotationProps['timestamp'];
+  timezone: AddAnnotationProps['timezone'];
+  value: { content: string; timestamp: number };
+  isOpen: boolean;
+  setIsOpen: Dispatch<SetStateAction<boolean>>;
+  popoverClassname?: string;
```

Comment:
Can you add back the tsdoc comments we have?
https://github.com/pyroscope-io/pyroscope/blob/53836ce6b2f4a5eb2debc6a2e03c0b39812bf488/webapp/javascript/pages/continuous/contextMenu/AddAnnotation.menuitem.tsx#L21-L28